python/cudf_polars/cudf_polars/containers/column.py

```diff
@@ -55,6 +57,65 @@ def __init__(
         self.name = name
         self.set_sorted(is_sorted=is_sorted, order=order, null_order=null_order)
 
+    @classmethod
+    def deserialize(
+        cls, header: Mapping[str, Any], frames: tuple[memoryview, plc.gpumemoryview]
```

Comment:
How thorough are we about typing here? Would it be OK to be more specific about the shape of `header`?

```
class ColumnOptions(TypedDict):
    is_sorted: plc.types.Sorted
    order: plc.types.Order
    null_order: plc.types.NullOrder
    name: str | None


class Header(TypedDict):
    column_kwargs: ColumnOptions,
    frame_count: int
```


I see we only currently make use of `column_kwargs`, but I think including everything that `serialize` outputs is best.

Reply:
Yeah, I think it would be nice to enhance the typing a bit. I gave this a shot, and tried to follow the convention of defining custom typing variables in `cudf_polars.typing` - @wence- Does that convention make sense in this case?

Reply:
I think that's reasonable.